items.json

```diff
@@ -70536,27 +70544,27 @@
     "weapons": []
   },
   {
-    "id": "crpg_helping_hand_v1_h0",
-    "baseId": "crpg_helping_hand_v1",
+    "id": "crpg_helping_hand_v2_h0",
+    "baseId": "crpg_helping_hand_v2",
     "name": "Helping Hand Steel Throwing Club",
     "culture": "Battania",
     "type": "Thrown",
-    "price": 5201,
-    "weight": 1.2,
+    "price": 2982,
+    "weight": 1.52,
     "rank": 0,
-    "tier": 9.232108,
+    "tier": 6.717087,
     "requirement": 0,
     "flags": [],
     "weapons": [
       {
         "class": "ThrowingAxe",
```

Comment:
they were given a bit more weight to be in line with other 1h blunt weapons. it may also help them avoid being stunlocked by other heavier weapons.  looking forward to the day when thrown weapons have usable melee stats!

Reply:
ok, so can we remove just that commit?

Reply:
reverted changes to helping hand, throwing mace and throwing club since they need to rebalanced with newly scaled and fit models on the spooky batch branch.